templates/history-downloads.php

```diff
@@ -36,6 +36,10 @@
 					<tr class="edd_download_history_row">
 						<?php
 						$price_id       = edd_get_cart_item_price_id( $download );
+						// Get price ID of product with variable prices included in Bundle
+						if ( $download['in_bundle'] && edd_has_variable_prices( $download['id'] ) ) {
```

Comment:
```suggestion
						if ( ! empty( $download['in_bundle'] ) && edd_has_variable_prices( $download['id'] ) ) {
```
I think we need to add the `empty` check as the `in_bundle` may not be set at all--I got an undefined index notice.

Once I made that change, the PR works as expected and does prevent all bundled items from showing for the download if they should not be.

Reply:
@robincornett, Thanks! Great suggestion.